docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,216 @@ task.download_outputs()
+
+### Important Details
+
+1. **ExaFOAM Benchmark**: This simulation replicates the MB9 micro-benchmark
+from the [ExaFOAM](https://exafoam.eu/benchmarks/) project. The benchmark
+focuses on capturing key flow physics and the simulation approach of a larger
+HPC challenge with fewer computational resources, providing a scalable and
+efficient simulation of a high-lift configuration.
+   
+2. **Machine Configuration**: We are using a powerful `c3d-standard-180`
+machine with 180 CPUs to ensure that the computation proceeds efficiently.
+You can adjust this depending on the available resources or your project's needs.
+   
+3. **Commands**: The list of `commands` involves several key OpenFOAM tasks
+like copying necessary configuration files, running mesh generation, and
+parallelizing the process using `decomposePar` and `snappyHexMesh`.
+
+4. **Parallelization**: We use `n_vcpus=128` in the simulation, which will
+allow OpenFOAM to run in parallel, greatly speeding up the computation.
+
+### Running the Simulation
+
+Running the script above will take some time, as OpenFOAM tasks like
+`snappyHexMesh` and `checkMesh` can be computationally intensive, especially
+with large meshes. The `task.wait()` command will block until the simulation
+is complete, after which the output files will be available.
+
+### Post-Simulation
+
+After the simulation completes, the output files will be downloaded locally by
+calling `task.download_outputs()`. The outputs will include simulation results
+like mesh quality reports, final results, and log files for each step in the process.
+
+To inspect the output, look under the `inductiva_output` folder where the
+results are downloaded:
+
+```bash
+ls -lasgo inductiva_output/task_id
+total 672
+  0 drwxr-xr-x  143     4576 Sep 19 16:06 .
+  0 drwxr-xr-x  400    12800 Sep 19 16:05 ..
+  0 drwxr-xr-x   11      352 Sep 19 16:06 0.orig
+  8 -rw-r--r--    1      626 Sep 19 16:06 Allclean
+ 16 -rw-r--r--    1     6998 Sep 19 16:06 Allrun
+  8 -rw-r--r--    1      991 Sep 19 16:06 COPYING
+ 48 -rw-r--r--    1    21547 Sep 19 16:06 README.md
+  0 -rw-r--r--    1        0 Sep 19 16:06 case.foam
+  0 drwxr-xr-x    8      256 Sep 19 16:06 constant
+  0 drwxr-xr-x   27      864 Sep 19 16:06 dynamicCode
+  0 drwxr-xr-x   13      416 Sep 19 16:06 figures
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor0
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor1
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor10
+    ...
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor126
+  0 drwxr-xr-x    4      128 Sep 19 16:06 processor127
+ 32 -rw-r--r--    1    14437 Sep 19 16:06 stderr.txt
+536 -rw-r--r--    1   272331 Sep 19 16:06 stdout.txt
+  0 drwxr-xr-x   30      960 Sep 19 16:06 system
+ 24 -rw-r--r--    1    11399 Sep 19 16:06 thumbnail.png
+```
+
+You can now perform post-processing on the results, just as you would if you had
+run the simulation locally.
+
```

Comment:
Can you also print the execution times?
And maybe try with two different machines. 
Try a bigger one....

Reply:
I will run the following executions (adding something about the times here after that)

c3d-standard-90 - nCores/n_vcpus = 90
c3d-standard-180 - nCores/n_vcpus = 180
c3d-standard-360 - nCores/n_vcpus = 360

c3d-standard-90 - nCores/n_vcpus = 45 (Disable hyperthreading)
c3d-standard-180 - nCores/n_vcpus = 90 (Disable hyperthreading)
c3d-standard-360 - nCores/n_vcpus = 180 (Disable hyperthreading)